src/APIdata.js

```diff
@@ -0,0 +1,40 @@
+const ApiUrl = 'https://cors-anywhere.herokuapp.com/https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
```

Comment:
- [ ] As the previous reviewer mentioned, the app is not working as expected even after your suggestions of waiting a few seconds and I can still see a few errors in the console. Please, kindly fix these issues in order to match the project requirements :wink:
![image](https://user-images.githubusercontent.com/81522435/146654056-bfb05485-71d4-4004-9f2f-7e8100bdceed.png)